frontend/admin/src/js/components/cmoAsset/CmoAssetPage.tsx

```diff
@@ -10,40 +10,41 @@ export const CmoAssetPage: React.FC = () => {
   return (
     <div>
       <header
-        data-h2-bg-color="b(linear-70[lightpurple][lightnavy])"
-        data-h2-padding="b(top-bottom, l) b(right-left, xl)"
+        data-h2-background-color="base(dt-linear)"
+        data-h2-padding="base(x2, 0)"
       >
-        <div data-h2-flex-grid="b(middle, expanded, flush, l)">
-          <div data-h2-flex-item="b(1of1) m(3of5)">
-            <h1
-              data-h2-font-color="b(white)"
-              data-h2-font-weight="b(800)"
-              data-h2-margin="b(all, none)"
-              style={{ letterSpacing: "-2px" }}
+        <div data-h2-container="base(center, large, x2)">
+          <div data-h2-flex-grid="base(center, 0, x2)">
+            <div data-h2-flex-item="base(1of1) l-tablet(3of5)">
+              <h1
+                data-h2-color="base(dt-white)"
+                data-h2-font-weight="base(700)"
+                style={{ letterSpacing: "-2px" }}
+              >
+                {intl.formatMessage({
+                  defaultMessage: "CMO Assets",
+                  description:
+                    "Heading displayed above the CMO Asset Table component.",
+                })}
+              </h1>
+            </div>
+            <div
+              data-h2-flex-item="base(1of1) l-tablet(2of5)"
+              data-h2-text-align="l-tablet(right)"
             >
-              {intl.formatMessage({
-                defaultMessage: "CMO Assets",
-                description:
-                  "Heading displayed above the CMO Asset Table component.",
-              })}
-            </h1>
-          </div>
-          <div
-            data-h2-flex-item="b(1of1) m(2of5)"
-            data-h2-text-align="m(right)"
-          >
-            <Link
-              href={paths.cmoAssetCreate()}
-              color="white"
-              mode="outline"
-              type="button"
-            >
-              {intl.formatMessage({
-                defaultMessage: "Create CMO Asset",
-                description:
-                  "Heading displayed above the Create CMO Asset form.",
-              })}
-            </Link>
+              <Link
+                href={paths.cmoAssetCreate()}
+                color="white"
+                mode="outline"
+                type="button"
+              >
+                {intl.formatMessage({
+                  defaultMessage: "Create CMO Asset",
+                  description:
+                    "Heading displayed above the Create CMO Asset form.",
+                })}
+              </Link>
+            </div>
           </div>
         </div>
       </header>
```

Comment:
Odd gap of white appearing between header and content.

<img width="694" alt="Screen Shot 2022-08-18 at 11 44 59 AM" src="https://user-images.githubusercontent.com/4127998/185446462-ecf39a5f-9bcd-4e4a-955f-228ecb717db1.png">